components/cluster/command/display.go

```diff
@@ -108,7 +115,21 @@ func displayDashboardInfo(clusterName string) error {
 	}
 
 	u.Path = "/dashboard/"
-	fmt.Println(u.String())
+
+	if tlsCfg != nil {
+		fmt.Println(
+			"Client certificate:",
+			color.CyanString(tidbSpec.Path(clusterName, spec.TLSCertKeyDir, spec.PFXClientCert)),
+		)
+		fmt.Println(
+			"Certificate password:",
+			color.CyanString(crypto.PKCS12Password),
```

Comment:
I don't we need to print the password to the output.

Reply:
This is a hard coded password, and user need that to import the certificate to their browsers. It's by intend to be hard coded as the PKCS#12 encryption itself is weak and the password is not providing much extra protection, but just a compatibility feature.